Example/Emission/ARRootViewController.m

```diff
@@ -84,6 +84,20 @@ - (void)viewDidLoad
   self.tableViewData = tableViewData;
 }
 
+- (void)viewDidAppear:(BOOL)animated
+{
+  [super viewDidAppear:animated];
+  
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    BOOL autoJump = [[NSUserDefaults standardUserDefaults] boolForKey:ARJumpStraightIntoStorybooks];
+    if (autoJump) {
+      id viewController = [ARStorybookComponentViewController new];
+      [self.navigationController pushViewController:viewController animated:YES];
+    }
+  });
+}
```

Comment:
Storybooks is only available for dev builds, right? If so, let’s not even compile this in for other builds:

```objc
#ifndef DEBUG
...
#endif